packages/api/__tests__/specs/author.ts

```diff
@@ -0,0 +1,202 @@
+import {MongoDBAdapter} from '@wepublish/api-db-mongodb'
+import {ApolloServerTestClient} from 'apollo-server-testing'
+import {createGraphQLTestClientWithMongoDB} from '../utility'
+import {
+  CreateAuthor,
+  AuthorInput,
+  AuthorList,
+  Author,
+  UpdateAuthor,
+  DeleteAuthor
+} from '../api/private'
+
+let testClientPublic: ApolloServerTestClient
+let testClientPrivate: ApolloServerTestClient
```

Comment:
did you use `let` on purpose here instead of `const` ? I assume that the content of `testClientPrivate` etc. won't be changing? But then again, that's just a minor remark :)

Reply:
@0815pascal this is just a declaration. The assignment happens 6 line below. It's not possible to use `const` here.

Reply:
Ok, all good then